pkg/ipc/ipc.go

```diff
@@ -283,30 +288,30 @@ func addFallbackSignal(p *prog.Prog, info []CallInfo) {
 	}
 	p.FallbackSignal(callInfos)
 	for i, inf := range callInfos {
-		info[i].Signal = inf.Signal
+		info.Calls[i].Signal = inf.Signal
 	}
 }
 
-func (env *Env) parseOutput(p *prog.Prog) ([]CallInfo, error) {
+func (env *Env) parseOutput(p *prog.Prog) (*ProgInfo, error) {
 	out := env.out
 	ncmd, ok := readUint32(&out)
 	if !ok {
 		return nil, fmt.Errorf("failed to read number of calls")
 	}
-	info := make([]CallInfo, len(p.Calls))
+	info := ProgInfo{Calls: make([]CallInfo, len(p.Calls))}
```

Comment:
Since it's an entity always passed by-pointer (as opposed to a small value type that is passed by value), it's more common and idiomatic to create a pointer right away rather then create a value and return `&info`. So please use `&ProgInfo{}` here.

Reply:
Fixed, PTAL.